content/en/monitors/notify/variables.md

```diff
@@ -367,23 +374,33 @@ To include **any** attribute or tag from a log, a trace span, a RUM event, a CI
 | CI Test         | `{{citest.attributes.key}}`                      |
 | Database Monitoring | `{{databasemonitoring.attributes.key}}`      |
 
-For any `key:value` pair, the variable `{{log.tags.key}}` renders `value` in the alert message.
+{{% collapse-content title="Example syntax usage" level="h4" %}}
+- For any `key:value` pair, the variable `{{log.tags.key}}` renders `value` in the alert message.
+- The `@` that prefixes all attributes is not included. For example, if a log monitor is grouped by `@http.status_code`, to include the error message or infrastructure tags in the notification message, use the variables:
+  
+  ```text
+  {{ log.attributes.[error.message] }}
+  {{ log.tags.env }}
+  ...
+  ```
 
-**Example**: If a log monitor is grouped by `@http.status_code`, to include the error message or infrastructure tags in the notification message, use the variables:
+  {{< img src="monitors/notifications/tag_attribute_variables.png" alt="Matching attribute variable syntax" style="width:90%;">}}
+- The message renders the `error.message` attribute of a chosen log matching the query, **if the attribute exists**.
+- If the tag is on an event use the following syntax:
 
-```text
-{{ log.attributes.[error.message] }}
-{{ log.tags.env }}
-...
-```
+  ```text
+  {{ event.tags.[dot.key.test] }}
+  ```
+
+
+{{% /collapse-content %}} 
 
-{{< img src="monitors/notifications/tag_attribute_variables.png" alt="Matching attribute variable syntax" style="width:90%;">}}
 
-The message renders the `error.message` attribute of a chosen log matching the query, **if the attribute exists**.
+#### Important notes
 
-<div class="alert alert-info"><strong>Note</strong>: If the selected event does not contain the attribute or the tag key, the variable renders empty in the notification message. To avoid missing notifications, do not use these variables for routing notification with <code>{{#is_match}}</code> handles.</div>
+- If the selected event does not include the attribute or tag key, the variable renders empty in the notification message. To prevent missing notifications, avoid using these variables for routing notification with `{{#is_match}}` handles.
```

Comment:
```suggestion
- If the selected event does not include the attribute or tag key, the variable renders empty in the notification message. To prevent missing notifications, avoid using these variables for routing notifications with `{{#is_match}}` handles.
```